microsetta_public_api/api/datasets.py

```diff
@@ -0,0 +1,18 @@
+from microsetta_public_api.utils._utils import jsonify
+from microsetta_public_api.resources_alt import get_resources
+from microsetta_public_api.config import schema
+
+
+def available():
+    resources = get_resources()
+    escape = {schema.metadata_kw}
+    dataset_key = 'datasets'
+
+    if dataset_key not in resources:
+        return jsonify([]), 200
```

Comment:
Should this be a 404?

Reply:
I think 200 with empty list is saying "I know you are looking for the names of datasets, but I have none, so here's any empty list".
Whereas 404 would be saying "You are looking for the names of datasets, but I cannot find anything like that, so here's an error".

I think the 200 case is more reflective of what is happening here.